tools/clang/unittests/HLSLExec/LongVectorOps.def

```diff
@@ -48,14 +49,36 @@ OP_DEFAULT(BinaryMath, Ldexp, 2, "ldexp", ",")
 OP_DEFAULT(Bitwise, And, 2, "", "&")
 OP_DEFAULT(Bitwise, Or, 2, "", "|")
 OP_DEFAULT(Bitwise, Xor, 2, "", "^")
-OP_DEFAULT_DEFINES(Bitwise, Not, 1, "TestUnaryOperator", "~",
-                   " -DFUNC_UNARY_OPERATOR=1")
 OP(Bitwise, LeftShift, 2, "", "<<", "", Default1, BitShiftRhs, Default3)
 OP(Bitwise, RightShift, 2, "", ">>", "", Default1, BitShiftRhs, Default3)
 
 OP_DEFAULT_DEFINES(Unary, Initialize, 1, "TestInitialize", "",
                    " -DFUNC_INITIALIZE=1")
 
+#define OP_CAST_DEFAULT_DEFINES(GROUP, SYMBOL) \
```

Comment:
The idea of the `OP_DEFAULT_DEFINES` macro is that it is the one that has a DEFINES parameter.

`OP_CAST_DEFAULT_DEFINES` doesn't have a DEFINES parameter.  

I think naming it `OP_CAST_DEFAULT` (since it uses Default validation and the Default input sets) would be more inline with the naming convention here.